src/native/libs/System.Security.Cryptography.Native/pal_evp_pkey_ml_dsa.h

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "opensslshim.h"
+#include "pal_compiler.h"
+#include "pal_types.h"
+
+/*
+Determines if the given signature algorithm is available.
+Returns 1 if available or 0 if not available.
+*/
+PALEXPORT int32_t CryptoNative_IsSignatureAlgorithmAvailable(const char* algorithm);
```

Comment:
Since the function doesn't say MLDsa in it, it should be in an MLDSA header.
openssl.h/c?